src/lib.rs

```diff
@@ -5188,6 +5233,55 @@ pub unsafe trait IntoBytes {
         Ok(())
     }
 
+    /// Writes a copy of `self` to the IO byte sink `dst`.
+    ///
+    /// This is a shorthand for `dst.write_all(self.as_bytes())`, and is useful
+    /// for interfacing with operating system byte sinks (files, sockets, etc.).
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
```

Comment:
If you want this to run, you could use a trick like we do here: https://github.com/google/zerocopy/blob/ace5bb0de76e150d2284984677892a5aaa34a36e/src/macros.rs#L711

Reply:
I don't think there's much sense in actually testing the IO. It'll make our tests more expensive to run, and will cause them to fail on platforms where file IO is stubbed out (e.g., wasm).

Reply:
Ack